README.md

```diff
@@ -182,9 +183,10 @@ Transifex a few hours after changes are merged. After they are translated, click
 _"i18n: Update translations from Transifex"_.
 
 Adding a new language:
+
 1. first add it to Transifex and translate the strings,
 2. create a folder with the new language code in `src/locales` and download the
-  translation file there,
+    translation file there,
```

Comment:
updated due to MegaLinter `Wrong amount of left-padding spaces(want multiple of 2)`